app/controllers/offer/plan_delivery.js

```diff
@@ -25,7 +25,7 @@ export default Ember.Controller.extend({
 
       new AjaxPromise("/gogovan_orders/calculate_price", "POST", this.session.get('authToken'), params)
         .then(data => {
-          var coupon = undefined;
+          var coupon;
           if(data.breakdown) {
             coupon = data.breakdown.coupon_discount;
```

Comment:
we can also declare variable here and that will be available throughout the function because of variable hoisting 
https://www.w3schools.com/js/js_hoisting.asp